### ISSUE TITLE
Help needed

### ISSUE THREAD

**Issue by dwitiyabhatt** [twilio/twilio-video-app-android#206]
Hello!

I had deployed twilio for basic 1-1 video calling.

Link : https://github.com/twilio/video-quickstart-android/tree/master/quickstartKotlin
 
I also have deployed user alert facility using firebase setup as per twilio example using this link :

https://github.com/twilio/video-quickstart-android/tree/master/exampleVideoInvite 

My question is : Is it possible to convert this application into video conferencing app? using existing controls? 

Or I need to do complete different setup with controls shown in example?

**Comment by Alton09** [comment]
Hi @dwitiyabhatt ! Thanks for reaching out. For a better collaborative video conferencing experience, I would suggest using the video app as part of this repository. Please read the [readme](https://github.com/twilio/twilio-video-app-android#getting-started) to learn how to quickly deploy the app.

**Comment by dwitiyabhatt** [comment]
Yes but, what is I want to use only some part of the code and not the other? For example, I don't want to use firebase auth and login stuff. I just need Video view with those multiple user recyclerview only. Is it possible? I want to avoid that status management also.

**Comment by Alton09** [comment]
Hi @dwitiyabhatt . I see, yes you can definitely pull out firebase auth pretty easily. In fact, if you look at the Community build variant of the app, you will see that password auth is used instead of firebase auth. If you want to completely bypass login, simply ignore the authenticator and navigate directly to the `RoomActivity` as part of the [SplashActivity](https://github.com/twilio/twilio-video-app-android/blob/760e7ac083cbdb995cbd516c3b9f872bf8ed41b4/app/src/main/java/com/twilio/video/app/ui/splash/SplashActivity.kt#L35). What exactly do you mean by status management?

**Comment by Alton09** [comment]
Closing this one for now due to inactivity.
